src/utils/helpers.js

```diff
@@ -0,0 +1,12 @@
+export function setButtonText(button, 
+    isLoading, 
+    loadingText = "Saving...", 
+    defaultText = "Save") {
+    const text = isLoading ? loadingText : defaultText;
```

Comment:
`    const text = isLoading ? loadingText : defaultText;
`

This is not needed because you have this check in line 7


And you have the texts as `loadingText` and `defaultText`



```js
export function setButtonText(
  button,
  isLoading,
  loadingText = "Saving...",
  defaultText = "Save"
) {
  if (isLoading) {
    button.textContent = loadingText;  // the loading text ( `"Saving..."` by default )
  } else {
    button.textContent = defaultText;  // the initial button text  ( `"Save"` by default )
  }
}

```